packages/atomic/src/components/search/smart-snippets/atomic-smart-snippet-suggestions/atomic-smart-snippet-suggestions.tsx

```diff
@@ -91,6 +92,7 @@ export class AtomicSmartSnippetSuggestions implements InitializableComponent {
 
     this.smartSnippetSuggestionListCommon = new SmartSnippetSuggestionCommon({
       id: this.id,
+      attributes: getAttributesFromLinkSlot(this.host, 'attributes'),
```

Comment:
I would prefer if this was a getter instead, to avoid race conditions.

For example:
```suggestion
      getAttributes: () => getAttributesFromLinkSlot(this.host, 'attributes'),
```

Reply:
Also, I would prefer if we used a different name than `attributes`, since the `atomic-smart-snippet`/`atomic-smart-snippet-suggestions` contain a lot of things that technically have attributes.

I think a name like "source-anchor-attributes" would work.

Reply:
Lastly, the jsdoc of the smart snippet and smart snippet suggestions should be updated to contain the `@slot` with a description. You can copy the one from `atomic-result-link` and adapt it, then tag `@srosenberg-coveo` for review.